File: src/main/java/jp/artan/japanesefoodmod/init/BiomeInit.java
package jp.artan.japanesefoodmod.init;

import jp.artan.japanesefoodmod.common.world.biome.CustomBiome;
import jp.artan.japanesefoodmod.common.world.biome.FruitForestBiome;
import jp.artan.japanesefoodmod.common.world.type.WorldTypeFruitsForest;
import net.minecraft.world.WorldType;
import net.minecraft.world.biome.Biome;

import java.util.ArrayList;
import java.util.List;

public class BiomeInit {
    public static List<CustomBiome> BIOMES = new ArrayList<CustomBiome>();

    // バイオーム
    public static final Biome FRUIT_FOREST_BIOME = new FruitForestBiome();

    // ワールドタイプ
    public static final WorldType FRUITS_FOREST_WORLD_TYPE = new WorldTypeFruitsForest();
}

File: src/main/java/jp/artan/japanesefoodmod/init/BiomeInit.kt
package jp.artan.japanesefoodmod.init

import jp.artan.japanesefoodmod.common.world.biome.CustomBiome
import jp.artan.japanesefoodmod.common.world.biome.FruitForestBiome
import jp.artan.japanesefoodmod.common.world.type.WorldTypeFruitsForest
import java.util.ArrayList

class BiomeInit {
    companion object {
        val BIOMES = ArrayList<CustomBiome>()

        // バイオーム
        val FRUIT_FOREST_BIOME = FruitForestBiome()

        // ワールドタイプ
        val FRUITS_FOREST_WORLD_TYPE = WorldTypeFruitsForest()
    }
}
